testcafe/tests/simple-flow.js

```diff
@@ -0,0 +1,69 @@
+import { Selector } from 'testcafe';
```

Comment:
Just add one more advanced test (i.e., with a hijack, anything you want; you can select a testfile example). Just make sure you check all possible fields in hijacks and hijack page. After that, we can merge and we can add tests also for the outdate-fix PR extra. Thanks!